circle.yml

```diff
@@ -12,158 +14,62 @@ dependencies:
     - firefox
 
   pre:
-    - sudo apt-get install graphicsmagick expect tightvncserver
-
     - |
-      if [ -e fxa-js-client ]; then
```

Comment:
The changes to this file are probably a lot easier to grok if using the split diff view.